### ISSUE TITLE
Update on progress

### ISSUE THREAD

**Issue by nvanselow** [QuickBase/babel-plugin-styled-components-css-namespace#27]
January is going to be the month of closing out some issues in this project. Unfortunately, I was unable to dedicate enough time to improvements here. Thank you to everyone who has reported issues. Please continue to report them or open PRs with fixes. My goal is to better be aware of issues coming into the project and responding more quickly going forward.

**Comment by TrevorBurnham** [comment]
@nvanselow FYI I recently created an alternative project that you might want to check out: https://github.com/TrevorBurnham/babel-plugin-namespace-styled-components/

**Comment by nvanselow** [comment]
Thanks for the heads up!

**Comment by nvanselow** [comment]
@TrevorBurnham - I took a look at your solution. It is sweet! It looks like we were working towards the same approach. I had been messing around with postcss as a better solution as well. You had some awesome tricks in there that helped me solve some issues I was facing.
- Brilliant use of `WeakSet`
- New helpers for rejecting keyframes, etc. in the latest version of babel-plugin-styled-components. I upgraded and pulled those in to simplify some of the checks.
- The deasync lib to deal with the promise in postcss
- Integration tests using the actual React and Styled Components libs. I hadn't considered doing that, but it's a great confirmation the final output is what we might expect.

One lib you might want to check out if you haven't is `postcss-parent-selector`. It does what we need it to without writing our own.

I'll add you to the final PR to ensure I've given appropriate credit. I'm still working on some cleanup and backwards compatibility with our previous API and will probably open a PR next weekend.
